File: app/src/main/java/com/skydoves/elasticviewsdemo/MainActivity.java
package com.skydoves.elasticviewsdemo;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

/**
 * Developed by skydoves on 2017-01-21.
 * Copyright (c) 2017 skydoves rights reserved.
 */

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void ElasticButtons(View v) {
        switch (v.getId()){
            case R.id.elasticbtn0 :
                startActivity(new Intent(getBaseContext(), ExampleActivity0.class));
                break;
            case R.id.elasticbtn1 :
                startActivity(new Intent(getBaseContext(), ExampleActivity1.class));
                break;
            case R.id.elasticbtn2 :
                startActivity(new Intent(getBaseContext(), ExampleActivity2.class));
                break;
        }
    }
}

File: app/src/main/java/com/skydoves/elasticviewsdemo/MainActivity.kt
/*
 * The MIT License (MIT)
 *
 * Copyright (c) 2017 skydoves
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in
 * all copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 * THE SOFTWARE.
 */
package com.skydoves.elasticviewsdemo

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.skydoves.elasticviewsdemo.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    val binding = ActivityMainBinding.inflate(layoutInflater)
    setContentView(binding.root)
  }

  fun buttons(v: View) {
    when (v.id) {
      R.id.elasticbtn0 -> startActivity(Intent(baseContext, ExampleActivity0::class.java))
      R.id.elasticbtn1 -> startActivity(Intent(baseContext, ExampleActivity1::class.java))
      R.id.elasticbtn2 -> startActivity(Intent(baseContext, ExampleActivity2::class.java))
    }
  }
}
